<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabGestioneCucina</class>
 <widget class="QTabWidget" name="tabGestioneCucina">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>771</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Struttura Alberghiera: Gestione Cucina</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="movable">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="ColazioneInCamara">
   <attribute name="title">
    <string>Colazione in camera</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item>
     <widget class="QLabel" name="labelIntroduzione">
      <property name="text">
       <string>Benvenuto, di seguito il menù dell'hotel per la colazione in camera:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupboxMenuColazioneInCamera">
      <property name="title">
       <string>Menù:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupboxDolce">
         <property name="title">
          <string>Dolce:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetDolceColazioneInCamera">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Piatto</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxSalato">
         <property name="title">
          <string>Salato:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetSalatoColazioneInCamera">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Piatto</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxBevande">
         <property name="title">
          <string>Bevande:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetBevandeColazioneInCamera">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Bevande</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetModificaMenu" native="true">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnConfermaModifiche">
         <property name="text">
          <string>Conferma</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAnnullaModifiche">
         <property name="text">
          <string>Annulla</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnModificaMenuColazioneInCamera">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Modifica Menù</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnTornarePaginaPrecedente">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Tornare indietro alla pagina precedente</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="Pranzo">
   <attribute name="title">
    <string>Pranzo</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item>
     <widget class="QLabel" name="labelIntroduzione_2">
      <property name="text">
       <string>Benvenuto, di seguito il menù dell'hotel per il pranzo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupboxMenuPranzo">
      <property name="title">
       <string>Menù:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupboxAntipastiPranzo">
         <property name="title">
          <string>Antipasti:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetAntipastiPranzo">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Antipasto</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxPrimiPranzo">
         <property name="title">
          <string>Primi:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetPrimoPranzo">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Primo</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxSecondiContorniPranzo">
         <property name="title">
          <string>Secondi e Contorni:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetSecondoContornoPranzo">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Secondo/Contorno</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxDolciBevandePranzo">
         <property name="title">
          <string>Dolci e Bevande:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetDolciBevandePranzo">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Dolce/Bevanda</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetModificaMenu_2" native="true">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAnnullaModifiche_2">
         <property name="text">
          <string>Annulla</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnConfermaModifiche_2">
         <property name="text">
          <string>Conferma</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnModificaMenuPranzo">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Modifica Menù</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnTornarePaginaPrecedente_2">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Tornare Indietro alla pagina precedente</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="Cena">
   <attribute name="title">
    <string>Cena</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item>
     <widget class="QLabel" name="labelIntroduzione_3">
      <property name="text">
       <string>Benvenuto, di seguito il menù dell'hotel per la cena:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupboxMenuCena">
      <property name="title">
       <string>Menù:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupboxAntipastiCena">
         <property name="title">
          <string>Antipasti:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetAntipastiCena">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Antipasto</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxPrimiCena">
         <property name="title">
          <string>Primi:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetPrimiCena">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Primo</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxSecondiContorniCena">
         <property name="title">
          <string>Secondi e Contorni:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetSecondiContorniCena">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Secondo/Contorno</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupboxDolciBevandeCena">
         <property name="title">
          <string>Dolci e Bevande:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treewidgetDolciBevandeCena">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <column>
             <property name="text">
              <string>Dolce/Bevanda</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ingredienti</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetModificaMenu_3" native="true">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAnnullaModifiche_3">
         <property name="text">
          <string>Annulla</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnConfermaModifiche_3">
         <property name="text">
          <string>Conferma</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnModificaMenuCena">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Modifica Menù</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnTornarePaginaPrecedente_3">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Tornare indietro alla pagina precedente</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
